project/src/components/cards-sorting/cards-sorting-list.tsx

```diff
@@ -0,0 +1,35 @@
+import cn from 'classnames';
+import {OffersSortTypes} from '../../const';
+
+type CardsSortingListProps = {
+  isSortingOpened: boolean;
+  selectedSortType: string
+  onChangeSortType: (currentCity: string) => void;
+}
+
+export default function CardsSortingList({isSortingOpened, selectedSortType, onChangeSortType}: CardsSortingListProps): JSX.Element {
```

Comment:
Мне кажется можно не удлинять имя компонента)
```suggestion
export default function SortingList({isSortingOpened, selectedSortType, onChangeSortType}: CardsSortingListProps): JSX.Element {
```